emails/templates/drafted.js

```diff
@@ -0,0 +1,33 @@
+const { addressBadge, action, infobox, base, link } = require('./shared')
+const { accountData } = require('../mock-utils')
+
+module.exports = function() {
+  return {
+    template: base({
+      title: 'Notifications',
+      subtitle: `Here are the notifications for the address ${addressBadge()} on {{date}}`,
```

Comment:
```suggestion
      subtitle: `Your account ${addressBadge()} received a notification on {{date}}.`,
```

Updated from Figma conversation

Reply:
Done in f81f019